libbeat/beat/beat.go

```diff
@@ -118,7 +122,7 @@ type BeatConfig struct {
 // Beat (e.g. packetbeat or topbeat). version is version number of the Beater
 // implementation. bt is Beater implementation to run.
 func Run(name, version string, bt Beater) error {
-	return newInstance(name, version, bt).launch(true)
+	return newInstance(name, version, bt).launch()
 }
 
 // instance contains everything related to a single instance of a beat.
```

Comment:
I did this because I want to remove `BT` from `Beat` to decouple them. When you call a function on `Beater` you pass in a `Beat` so you have no need for a reference to `BT` (it's almost circular).

If they are decoupled you can easily use the same `Beat` object with multiple instances of `Beater`'s (i.e. multiple Beater's in the same process sharing a common `Beat`.)